tests/phpunit/util/wpml-pb-test-case2.php

```diff
@@ -0,0 +1,191 @@
+<?php
+
+abstract class WPML_PB_TestCase2 extends \OTGS\PHPUnit\Tools\TestCase {
```

Comment:
@adrianosferreira Most of the code in this class is not used. Please review if you really need this class at all.